internal/slicer/report.go

```diff
@@ -70,3 +69,33 @@ func (r *Report) Add(slice *setup.Slice, fsEntry *fsutil.Entry) error {
 	}
 	return nil
 }
+
+// Mutate updates the FinalHash and Size of an existing path entry.
+func (r *Report) Mutate(fsEntry *fsutil.Entry) error {
+	relPath, err := r.sanitizePath(fsEntry.Path, fsEntry.Mode.IsDir())
```

Comment:
I think that the path should not be called `relPath` but `sanPath` or `path`, thoughts?

Reply:
`path` sounds good, yeah.